trade-finance-manager-ui/scripts/moj-filter-hide.ts

```diff
@@ -0,0 +1,91 @@
+import $ from 'jquery';
```

Comment:
Last comment - is the jquery import required purely because of the MOJ package?

Reply:
Yep, the [MOJ component expects elements with jQuery functions](https://github.com/ministryofjustice/moj-frontend/blob/1a02b936d8ff76279200fcc6185df2dce87bf5da/src/moj/components/filter-toggle-button/filter-toggle-button.js#L66) (e.g. `addClass` and `removeClass`).

I tried to achieve this with `document.querySelector` but this returns a standard DOM Element without these methods, so I've used jQuery to provide them rather than adding a wrapper around Element.

I've just added a comment to the script to summarise this